project/chrome-extension/script-two.js

```diff
@@ -18,4 +20,17 @@ function wordFoundAndIntro(){
    const intro = document.getElementById("wordTitle");
     //Adds the word of the id to the wordTitle element
    intro.innerText = "See the latest information about " + localStorage.getItem("rdoIdVarKey");
-}
\ No newline at end of file
+}
+
+// Add information aboout word to info.html
+function createDescription(){
+  const description = document.getElementbyId("description");
+  //description.innerText = ...;
+}
+
+// Add sources, such as link or references, to the information used for the
+// description of the word.
+function addSources(){
+  const source = document.getElementById("sources");
+  //source.innerText = _______________
```

Comment:
Same as above

Reply:
We left these comments on purpose so that @SamRod33 and @SofiaPSU could know where the information from the JSON should go. After those were implemented they would be removed.